### PR TITLE
receive page new layout setup

### PR DESCRIPTION
**Pull request by Sirmorrison**
### Resolves

Issue #96 

### What's new

This issue migrates the receive page to the new project layout

### Relevant screenshots or logs

ezgif com-video-to-gif(3)

### CODE REVIEW

File: ui/page.go [hunk 1]
```diff
@@ -100,12 +112,47 @@ func (page pageCommon) LayoutWithWallets(gtx *layout.Context, body layout.Widget
...
+	}
+
+	accounts := make([]decredmaterial.TabItem, len(page.info.Wallets[*page.selectedWallet].Accounts))
+	for i := range page.info.Wallets[*page.selectedWallet].Accounts {
+		accounts[i] = decredmaterial.TabItem{
+			Label: page.theme.Body1(page.info.Wallets[*page.selectedWallet].Accounts[i].Name),
+		}
+	}
+	page.accountsTab.SetTabs(accounts)
+	page.accountsTab.Position = decredmaterial.Top
```
Comment (oluwandabira) [inline ui/page.go]: This should be set in `addPages`

File: ui/recieve_page.go [hunk 2]
```diff
@@ -1,225 +1,313 @@
...
+	}
+	decredmaterial.Card{}.Layout(p.gtx, selectedDetails)
+}
+
+func (p *receivePage) qrCodeAddressColumn(common pageCommon) {
+	p.addrs = common.info.Wallets[*common.selectedWallet].Accounts[*common.selectedAccount].CurrentAddress
+	qrCode, err := qrcode.New(p.addrs, qrcode.Highest)
 	if err != nil {
-		win.err = err.Error()
+		log.Debug("Error generating address qrCode: " + err.Error())
```
Comment (oluwandabira) [inline ui/recieve_page.go]: log.Error

File: ui/recieve_page.go [hunk 3]
```diff
@@ -1,225 +1,313 @@
...
+type receivePage struct {
+	pageContainer layout.List
+	gtx           *layout.Context
+
+	isNewAddr, isInfo bool
+	addrs             string
+
+	newAddrBtn, gotItBtn                                 decredmaterial.Button
+	copyBtn, infoBtn, moreBtn                            decredmaterial.IconButton
+	copyBtnW, infoBtnW, moreBtnW, gotItBtnW, newAddrBtnW widget.Button
```
Comment (oluwandabira) [inline ui/recieve_page.go]: "gotIt" is not a good variable name

**Review by oluwandabira — CHANGES_REQUESTED** [review]

File: ui/recieve_page.go [hunk 4]
```diff
@@ -1,225 +1,313 @@
...
+				p.infoDiag()
+			}
+		}),
 	)
 }
 
-func (win *Window) qrCodeAddressColumn() {
-	addrs := win.walletInfo.Wallets[win.selected].Accounts[win.selectedAccount].CurrentAddress
-	qrCode, err := qrcode.New(addrs, qrcode.Highest)
+func (p *receivePage) selectedAcountColumn(common pageCommon) {
```
Comment (oshorefueled) [inline ui/recieve_page.go]: There's a typo on the method name.

File: ui/decredmaterial/icon.go [hunk 5]
```diff
@@ -7,7 +7,7 @@ import (
 	"image"
 	"image/color"
 	"image/draw"
-	_ "image/png"
+	_ "image/png" // fixes unknown format error when trying to decode png images
```
Comment (oshorefueled) [inline ui/decredmaterial/icon.go:10]: this comment should be rephrased. It makes png a decodable format, it doesn't fix an error.

**Review by oshorefueled — APPROVED** [review]

File: ui/decredmaterial/icon.go (hunk 5 above)
Comment (oshorefueled) [inline ui/decredmaterial/icon.go]: this comment should be rephrased. It makes png a decodable format, it doesn't fix an error.

**Review by oshorefueled — CHANGES_REQUESTED** [review]

**Review by oshorefueled — DISMISSED** [review]

**Review by oluwandabira — APPROVED** [review]

**Pull request merged**